cmd/server/main.go

```diff
@@ -57,18 +58,19 @@ func run() error {
 		repo, err = storage.NewFileStorage(
 			conf.FileStoragePath,
 			conf.StoreInterval,
-			conf.Restore)
+			conf.Restore,
+			mylog)
 		if err != nil {
 			return fmt.Errorf("error creating file repo: %w", err)
 		}
 	} else {
 		repo = storage.NewMemStorage()
 	}
-	h, err := handlers.NewMetricsHandler(repo)
+	h, err := handlers.NewMetricsHandler(repo, mylog)
```

Comment:
Если ты помещаешь логгер в разные слои, то хорошей идеей может быть добавление поля в логгер, которое указывает имя компонента, который пишет логи. Что-то типа

```suggestion
	h, err := handlers.NewMetricsHandler(repo, mylog.With(zap.String("component", "handlers")))
```

Reply:
done